evm_loader/program/Cargo.toml

```diff
@@ -56,6 +56,7 @@ cfg-if = { version = "1.0" }
 log = { version = "0.4", default-features = false, optional = true }
 maybe-async = "0.2.10"
 async-trait = { version = "0.1.79", optional = true }
+allocator-api2 = "0.2.16"
```

Comment:
In this file:
- the `serde_bytes` dependency can be removed;
- the `[dev-dependencies]` section can be removed completely, since those dependencies are no longer used.

Reply:
done